lib/components.dart

```diff
@@ -0,0 +1,21 @@
+export 'src/components/component.dart';
+export 'src/components/base_component.dart';
+export 'src/components/isometric_tile_map_component.dart';
+export 'src/components/nine_tile_box_component.dart';
+export 'src/components/parallax_component.dart';
+export 'src/components/particle_component.dart';
+export 'src/components/position_component.dart';
+export 'src/components/sprite_animation_component.dart';
+export 'src/components/sprite_batch_component.dart';
+export 'src/components/sprite_component.dart';
+export 'src/components/text_box_component.dart';
+export 'src/components/text_component.dart';
```

Comment:
Where is timer component?

Reply:
I tought as timer as a possible sub package by itself. 
`import 'package:flame/timer.dart'` imports the component and the `Timer` class.

Reply:
I think this would break the logic isn't it? Since all the components are here, I think that one should be too

Reply:
Changing that